cypress/integration/user_roles.js

```diff
@@ -0,0 +1,42 @@
+import LoginPage from '../objects/loginPage'
+import TimeLoggingPage from '../objects/timeLoggingPage';
+
+describe('Sourcebooks login', function() {
+
+    const loginPage = new LoginPage()  
+    const timeLogging = new TimeLoggingPage();    
+
+    it('Should display validation for empty user after attempted loggin', function () {
+        
+        loginPage.visit()
+        loginPage.getValidationError().should('not.visible')
+        loginPage.submitButton().click();
+        loginPage.getValidationError().should('be.visible')
+    })
+
+    const roles= [['User', 1], ["Team Lead", 2], ['Manager', 5], ['Accountant', 5], ['Admin', 6]]
```

Comment:
is there any reason for team lead to be enclosed in double quotes and rest of the names in single quotes?